### PR TITLE
build: create docker file

### PR DESCRIPTION
**Pull request by ologbonowiwi**
This aims to solve #53.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY requirements.txt .
+
+RUN pip install -r requirements.txt
```
Comment (joshiayush) [inline Dockerfile:7]: Can you change this to `pip install --no-cache-dir -r requirements.txt`

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM python:3.8
+
+WORKDIR /app
+
+COPY requirements.txt .
+
```
Comment (joshiayush) [inline Dockerfile:6]: It would be good if you can remove this empty line here.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+RUN pip install -r requirements.txt
+
+COPY . .
```
Comment (joshiayush) [inline Dockerfile:9]: I don't think that we need to copy everything to the `/app` directory just copy the `inb` directory that's in the project's root directory. Change this directive to:

```Dockerfile
COPY inb /app
```

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+CMD [ "python", "app.py" ]
```
Comment (joshiayush) [inline Dockerfile:11]: See the entry file is located inside of the `inb` directory which is [this](https://github.com/joshiayush/inb/blob/main/inb/inb.py). So please change this directive to:

```Dockerfile
CMD ["python", "inb/inb.py"]
```

Comment (joshiayush) [inline Dockerfile:11]: And please add a newline character at the end of this file :+1:

**Review by joshiayush — CHANGES_REQUESTED** [review]

**Comment by joshiayush** [comment]
@ologbonowiwi Can you also update the `README` to include the guide on build and usage of the docker setup?

**Comment by ologbonowiwi** [comment]
All comments addressed.

**Comment by joshiayush** [comment]
Thanks @ologbonowiwi :+1: I'll be waiting for more contributions from your side

File: README.md [hunk 5]
```diff
@@ -39,14 +39,40 @@ The tool is open source and available on GitHub, so users can contribute to the
...
 
+<div align="right">
+  <a href="#top">
+  
+  ![Back to top][back_to_top]
+  
+  </a>
+</div>
+
+## Docker usage
```
Comment (joshiayush) [inline README.md:58]: I didn't notice but I think we don't need the term "Docker usage" here, just "Docker" would be fine enough :+1:

**Pull request merged**